astropy/io/votable/tests/converter_test.py

```diff
@@ -65,6 +66,42 @@ def test_unicode_mask():
     assert c.output("Foo", True) == ''
 
 
+def test_unicode_as_char():
+    config = {'verify': 'exception'}
+    #with catch_warnings(exceptions.W55) as w:
```

Comment:
@tomdonaldson , if you have no plans to re-enable `catch_warnings` here, please remove the code.

Reply:
@pllim Thank you for the review!  These are all good comments and I will make the suggested changes.